pkgs/applications/version-management/sourcehut/core.nix

```diff
@@ -16,7 +16,7 @@ buildPythonPackage rec {
 
   node_modules = fetchNodeModules {
     src = "${src}/srht";
-    nodejs = nodejs-11_x;
+    nodejs = nodejs;
     sha256 = "0axl50swhcw8llq8z2icwr4nkr5qsw2riih0a040f9wx4xiw4p6p";
```

Comment:
The hash probably changes with different Node.js versions so it would be better to pin it to a certain release and change it everytime the version reaches EOL.

Reply:
Nevermind it doesn't seem to have changed.